great_expectations/rule_based_profiler/profiler.py

```diff
@@ -129,10 +132,12 @@ def profile(
         self,
         *,
         expectation_suite_name: Optional[str] = None,
+        include_citation: bool = True,
     ) -> ExpectationSuite:
         """
         Args:
             :param expectation_suite_name: A name for returned Expectation suite.
+            :param include_citation: Whether or not to include Profiler's config in metadata
```

Comment:
This is wordier, but maybe something like
```suggestion
            :param include_citation: Whether or not to include the Profiler config in the metadata for the ExpectationSuite produced by the Profiler
```

Reply:
Definitely like this a whole lot more. I've made this change!